src/app/shared/components/template/components/number-selector/number-selector.component.ts

```diff
@@ -48,42 +50,58 @@ export class TmplNumberComponent extends TemplateBaseComponent implements ITempl
     this.height = getStringParamFromTemplateRow(this._row, "height", "short");
     this.height = this.height === "short" || "normal" ? this.height : "short";
     if (this.category_list) {
-      this.valuesFromCategoryList = this.category_list.replace(/\s/g, "").split(";");
-      this.first_display_term = getNumberParamFromTemplateRow(this._row, "first_display_term", 1) - 1;
+      this.valuesFromCategoryList = this.category_list.replace(/\s/g, "").split(",");
+      this.first_display_term =
+        getNumberParamFromTemplateRow(this._row, "first_display_term", 1) - 1;
       this.displayValue = this.valuesFromCategoryList[this.first_display_term];
+      this._row.value = this.displayValue;
     } else {
       this.min_value = getNumberParamFromTemplateRow(this._row, "min_value", 0);
       this.max_value = getNumberParamFromTemplateRow(this._row, "max_value", 6);
       this.category_size = getNumberParamFromTemplateRow(this._row, "category_size", 1);
       this.displayValue = this.min_value;
+      this._row.value = this.displayValue;
     }
   }
 
   increment(param: "gt" | "lt") {
-    return this.category_list ?
-      (this.checkIfContainsNextArrayItem(this.valuesFromCategoryList, param) ?
-        this.updateData(param) : null)
-      : (param === "gt" ? (this.displayValue + this.category_size <= this.max_value ?
-        (this.category_size === 1 ? this.displayValue++ : (this.displayValue += this.category_size)) : null) :
-        (this.displayValue > this.min_value - this.category_size && this.displayValue - this.category_size >= this.min_value ?
-        (this.category_size === 1 ? this.displayValue-- : this.displayValue -= this.category_size) : null));
+    this.triggerActions("changed");
+    return this.category_list
```

Comment:
I reckon if statements here would be easier to read than this many nested ternary operators